agogosml_cli/cli/templates/pipeline/azure-ci-agogosml-pipeline.jsonnet

```diff
@@ -85,7 +85,7 @@ local repository = import 'pipeline-repository.libsonnet';
                         "inputs": {
                             "containerregistrytype": "Azure Container Registry",
                             "dockerRegistryEndpoint": "",
-                            "azureSubscriptionEndpoint": std.extVar('SUBSCRIPTION_ID'),
```

Comment:
I will probably try to move our json to yaml.
they might have fixed the issue with internal GUIDs, worth a try